frontend/src/common/queries/wheresMyGene.ts

```diff
@@ -1101,6 +1102,7 @@ function useWMGFiltersQueryRequestBody(
         sex_ontology_term_ids: sexes,
         tissue_ontology_term_ids: tissues,
         publication_citations: publications,
+        cell_type_ontology_term_ids: filteredCellTypeIds,
```

Comment:
this doesn't seem to be working - not sure if there's something i'm missing but it seems like either it's not being sent to the WMG call